IdentityCore/src/webview/systemWebview/ios/MSIDSafariViewController.m

```diff
@@ -66,12 +66,6 @@ - (instancetype)initWithURL:(NSURL *)url
             __auto_type config = [SFSafariViewControllerConfiguration new];
             _safariViewController = [[SFSafariViewController alloc] initWithURL:url configuration:config];
         }
-#if !TARGET_OS_MACCATALYST
```

Comment:
what is the reason for this change?

Reply:
This method is already deprecated when I build it with CMake. So I removed it and seems not causing any issue for build in common repo there.  Let me know if you have concerns. Thanks!

Reply:
We still support iOS 10 where alternative method is unavailable, so we should keep this.

Reply:
Reverted this change.